hw/ip/dma/data/dma_testplan.hjson

```diff
@@ -245,7 +245,7 @@
               - Check if PLIC interrupt is asserted when memory_buffer_auto_increment is enabled
               '''
         stage: V2
-        tests: ["dma_handshake_mem_buffer_overflow"]
+        tests: ["dma_handshake_stress"]
```

Comment:
@alees24: Could you please confirm that the `dma_handshake_stress` test includes random configurations that exercise the memory buffer threshold and limit behavior?

Reply:
Thx, added `dma_mem_limit`